napari/_qt/layer_controls/qt_labels_controls.py

```diff
@@ -196,7 +200,7 @@ def __init__(self, layer):
             'napari:activate_fill_mode',
             self.fill_button,
             extra_tooltip_text=trans._(
-                "Toggle with {shortcut}",
+                "\n(or {shortcut})",
                 shortcut=Shortcut("Control"),
             ),
```

Comment:
At present, this `Control` keybind is defunct. 
Was removed here:
https://github.com/napari/napari/pull/3072/files
It will be possible to return this when https://github.com/napari/napari/pull/5018 goes in, but via normal shortcuts and not this hard-coded. So I suggest deleting.

Reply:
Deleted